deploy/helm/flightctl/charts/keycloak/templates/keycloak-db-pvc.yaml

```diff
@@ -1,16 +1,19 @@
+{{ if and (eq (.Values.global).target "standalone") (eq (.Values.global.auth).type "builtin") }}
 kind: PersistentVolumeClaim
 apiVersion: v1
 metadata:
   name: keycloak-db
-  namespace: {{ .Values.db.namespace }}
+  namespace: {{ .Release.Namespace }}
   annotations:
-{{ toYaml .Values.db.pvc.annotations | indent 4 }}
+    kubernetes.io/reclaimPolicy: Delete
   labels:
-{{ toYaml .Values.db.pvc.labels | indent 4 }}
```

Comment:
Hmmm those toYaml files, I suspect are necessary, to deploy on the internal cloud there are some annotations/labels that we need to set.

Reply:
I also noticed that in other places I was looking at `.Values.appCode` but it should have been `.Values.global.appCode`

All should be fixed now

Reply:
Yes @fzdarsky and I were talking about changing this into generic labels, because this is something very specific to our cloud. But we could do on a 2nd round later. This is all round other places I believe.

Reply:
I agree, lets move to generic labels in a follow-up